### PR TITLE
add LICENSE

### LINKED ISSUE #23: Add a LICENSE file, please.

### PR DESCRIPTION
**Pull request by joshgoebel**
Resolves #23

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,13 @@
+            DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
+                    Version 2, December 2004
+
+ Copyright (C) 2004 Sam Hocevar <sam@hocevar.net>
```
Comment (haltman-at) [inline LICENSE:4]:
```suggestion
 Copyright (C) 2004 Sam Popischil <sam@changegiving.com>
```

**Review by haltman-at — COMMENTED** [review]
Seems good to me!  Just, the copyright line should be changed, right?

**Comment by joshgoebel** [comment]
Ha I think I just pasted it as is, duh.  BetteR?

**Review by haltman-at — APPROVED** [review]
Seems good!

**Pull request merged**